### ISSUE TITLE
Please fully document this namespace

### ISSUE THREAD

**Issue by benstevens48** [MicrosoftDocs/winrt-api#839]
There are a number of methods and operator overloads missing from the documentation of this namespace. Also for C++/WinRT the names of the structs are all different. Any chance it could be fully documented?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 96ca057d-145d-0460-ab1f-398399bf53e1
* Version Independent ID: dc5a4232-268c-f80d-f20b-fe2b4cb5e516
* Content: [Windows.Foundation.Numerics Namespace - Windows UWP applications](https://docs.microsoft.com/en-us/uwp/api/Windows.Foundation.Numerics)
* Content Source: [https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.foundation.numerics.yml&version=GBlive&_a=contents](https://cpubwin.visualstudio.com/DefaultCollection/windows-uwp/_git/winrt-api-build?path=%2fwinrt-api-build%2fwindows.foundation.numerics.yml&version=GBlive&_a=contents)
* Product: **uwp**
* Git Login: **hickeys**
* Microsoft Alias: **hickeys**

**Comment by stevewhims** [comment]
@benstevens48, for the second part of your feedback, I've restored from archive (and back into the fresh content) the windowsnumerics topics such as this one: https://review.docs.microsoft.com/previous-versions/windows/desktop/numerics_h/windowsnumerics-h-apis-portal?branch=master.

I've also added links to those topics from this set of topics. That covers the second part of your feedback, I think.

~Steve
